### ISSUE TITLE
HFM.NET appears in ALT+Tab even though it is running in notification tray (not minimized)

### ISSUE THREAD

**Issue by PantherX** [harlam357/hfm-net#335]
## Your Environment

* **HFM.NET Software version**: 0.9.17.1040
* **Operating System**: Windows 10 2004 64-bit 

----
## Expected Behavior

I expect that if HFM.NET is minimized to the notification tray, it will not appear in the list of applications that are opened when I use `ALT+Tab` to switch to other open applications.

----
## Current Behavior

Even when minimized to the notification tray, it appears in the list right at the end. Below is the screenshot showing this
![HFM NET](https://user-images.githubusercontent.com/5349122/91624259-588a3a00-e9f3-11ea-836a-ec70e26f326f.jpg)

----
## Possible Solution (Optional)

Like other Windows applications, e.g. Slack where it appears in `ALT+Tab` if it is open on the desktop or minimized to the taskbar but doesn't show up if it is closed to the notification tray. I would suggest that the options available mimic these from Slack which is really intuitive and can support virtually all options (auto start, manually start, start in notification tray, closing window doesn't exit the application, etc.)
![image](https://user-images.githubusercontent.com/5349122/91624372-144b6980-e9f4-11ea-8e4c-3029648c513f.png)

----
## Steps To Reproduce

1. Install HFM.NET
2. Double click the icon in the notification tray
3. Minimize the window so it returns back to notification tray
4. Press `ALT+Tab` and view HFM.NET in the list of applications to switch to.

----
## Context

HFM.NET doesn't follow the behaviors from other applications so I regularly end up accidentally clicking on the X window thinking that it will close to the notification tray but instead it has simply exited which means I have to start it up again to ensure that it captures data from my clients. I like consistency across applications that I use as it means I can be lazy and not remember what application behaves differently 🤣 

----

**Comment by harlam357** [comment]
Reading you loud and clear.  More options.  The means to remove HFM from the task bar is a quite simple, built-in function, so to accommodate this I'm going to need to research.  Seems like removing the app from the ALT+TAB list would be a baked-in thing.  Nothing I've ever noticed or really cared about.  I'm currently reworking the UI stack, so when I get to the right point, I'll see what I can do.  Not high priority but I concur this workflow could be better.

Just to recap, I'm reading two requests:

1) Option to [X] minimizes instead of terminating the application
2) When minimized in system tray ("notification area" seems to be the current terminology), don't show in ALT+TAB.

Auto start, run minimized, and minimize to system tray (notification area) are already options.  I've found another open source .NET application that seems to exhibit the behavior for 2), so I might be able to learn something from that code base.

**Comment by PantherX** [comment]
Yes, that's correct that they are two requests and you have summed it nicely!

Completely understand that you're busy with reworking the UI stack and I look forward to it 😃
